packages/apps-service/src/modules/apps/resolver.ts

```diff
@@ -101,13 +102,22 @@ export default <IResolvers<App, IAppsContext>>{
       if ( !app ) {
         throw new Error( 'App not found' );
       }
+      const database = {
+        name: databaseName,
+        description,
+        permissions,
+      };
+
+      if ( isEmpty( database.permissions ) ) {
+        /* TODO: Add default users from the app permission model */
+        database.permissions = { admins: [ 'user:' + app.ownerId ], users: [ 'user:' + app.ownerId, 'op-users' ] }
+      }
 
       try {
         /* Create the database on couchdb */
-        await createDatabase( databaseName );
+        await createDatabase( database.name );
```

Comment:
@deshmukhmayur How we are setting the description?

Reply:
This is the service call to couchdb. The description is not needed for this.